content/blog/part-8-agentic-ai-and-qdrant-building-semantic-memory-with-mcp-protocol.md

```diff
@@ -0,0 +1,217 @@
+---
+title: "Part 8: Agentic AI and Qdrant: Building Semantic Memory with MCP Protocol"
```

Comment:
Suggestion:
sentence case for the title. However, you may want to wait for Dale's feedback before modifying the title.

My suggestion:
Part 8: Agentic AI and Qdrant: Building Semantic Memory with MCP protocol